src/SophiApp/Localizations/IT.xaml

```diff
@@ -3,26 +3,26 @@
     <system:String x:Key="Conditions.OsBuildVersion.Supported">SophiApp supporta Windows 10 2004/20H2/21H1/21H2,</system:String>
     <system:String x:Key="Conditions.OsBuildVersion.SupportedInsider">Windows 11 21H2, 22H2 &amp; Windows 11 Insider Preview.</system:String>
-    <system:String x:Key="Conditions.OsFilesNotCorrupted.IsCorrupted">Alcuni componenti di Windows potrebbero essere rotti.</system:String>
-    <system:String x:Key="Conditions.OsBuildVersion.RunUpdate">Esegui Windows Update per aggiornare il sistema operativo all'ultima versione.</system:String>
-    <system:String x:Key="Conditions.OsFilesNotCorrupted.ReinstallOs">Si prega di reinstallare il sistema operativo.</system:String>
-    <system:String x:Key="Conditions.OsIsInfected">Il tuo sistema potrebbe essere infettato per una backdoor in Win 10 Tweaker.</system:String>
-    <system:String x:Key="Conditions.OsIsInfected.More">Scopri di più:</system:String>
+    <system:String x:Key="Conditions.OsFilesNotCorrupted.IsCorrupted">Alcuni componenti di Windows potrebbero essere danneggiati.</system:String>
+    <system:String x:Key="Conditions.OsBuildVersion.RunUpdate">Esegui Windows Update per aggiornare il sistema operativo alla versione più recente.</system:String>
+    <system:String x:Key="Conditions.OsFilesNotCorrupted.ReinstallOs">Reinstalla il sistema operativo.</system:String>
+    <system:String x:Key="Conditions.OsIsInfected">Questo sistema potrebbe essere stato infettato tramite una backdoor in Win 10 Tweaker.</system:String>
+    <system:String x:Key="Conditions.OsIsInfected.More">Maggiori info:</system:String>
     <system:String x:Key="Conditions.OsIsInfected.Link">https://itnan.ru/post.php?c=1&amp;p=557388</system:String>
-    <system:String x:Key="Conditions.RiskAcceptance">Sono consapevole dei rischi e mi assumo la responsabilità di potenziali danni</system:String>
+    <system:String x:Key="Conditions.RiskAcceptance">Sono consapevole dei rischi e mi assumo la responsabilità dei potenziali danni</system:String>
     <system:String x:Key="Conditions.DefenderIsBroken">Microsoft Defender rimosso dal sistema</system:String>
     <system:String x:Key="Conditions.Windows10DebloaterNotUsed.StabilityCompromised">La stabilità di Windows potrebbe essere stata compromessa da</system:String>
-    <system:String x:Key="Conditions.Windows10DebloaterNotUsed.SycnexScript">Windows10Debloater script di Sycnex. Reinstalla windows se necessario.</system:String>
-    <system:String x:Key="Conditions.OsUpdateBuildRevision.MinimumVersion">L'attuale versione di Windows non é aggiornata.</system:String>
+    <system:String x:Key="Conditions.Windows10DebloaterNotUsed.SycnexScript">Windows10Debloater script di Sycnex. Se neecssario reinstalla Windows.</system:String>
+    <system:String x:Key="Conditions.OsUpdateBuildRevision.MinimumVersion">Questa versione di Windows non é aggiornata.</system:String>
```

Comment:
i prefer L'attuale(more coherent with line 12)

Reply:
As for line 12 I beleieve that l'attuale" is quite generic and not so used in italian in software environment.